test/models/report_test.rb

```diff
@@ -126,6 +126,38 @@ def setup
       assert_equal({}, report.metrics)
     end
 
+    test '#metrics loads string with HashWithIndifferentAccess' do
+      report = Report.new
+      report[:metrics] = '---
+        events: !ruby/hash:ActiveSupport::HashWithIndifferentAccess
+          total: 1
+          success: 1
+          failure: 0'
+      assert_nothing_raised { report.metrics }
```

Comment:
Would you say the `assert_nothing_raised` is kinda overkill here?

Reply:
I like being explicit, so I'm good with it.